<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IHMPikawa</class>
 <widget class="QMainWindow" name="IHMPikawa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pikawa 2022</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="ecrans">
      <widget class="QWidget" name="accueil">
       <property name="styleSheet">
        <string notr="true">background:rgb(193, 125, 17);</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="layoutEnteteAccueil">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelTitreApplicationAccueil">
            <property name="font">
             <font>
              <family>Trebuchet MS</family>
              <pointsize>14</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:rgb(238, 238, 236);</string>
            </property>
            <property name="text">
             <string>PIKAWA</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight|Qt::AlignVCenter">
           <widget class="QLabel" name="labelEtatCafetiereAccueil">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="ressources.qrc">:/cafetiere-off.png</pixmap>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layoutCentralAccueil">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="layoutMenuAccueil">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="boutonInformations">
              <property name="font">
               <font>
                <family>Trebuchet MS</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Informations</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="boutonEntretien">
              <property name="font">
               <font>
                <family>Trebuchet MS</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Entretien</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="boutonParametres">
              <property name="font">
               <font>
                <family>Trebuchet MS</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Param√®tres</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="information"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">background:rgb(233, 185, 110);</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
